demos/sock-shop/sock-shop-loadgen.yaml

```diff
@@ -28,10 +28,10 @@ spec:
 
       initContainers:
       - name: wait-sock-shop
-        image: alpine:3.6
+        image: gcr.io/pixie-oss/pixie-dev-public/curl:multiarch-7.87.0
         # yamllint disable rule:indentation
-        command: ['sh', '-c', 'set -x;  apk add --no-cache curl;
-          until timeout -t 2 curl -f "${SOCK_SHOP_HEALTH_ADDR}"; do
+        command: ['sh', '-c', 'set -x;
+          until timeout 2 curl -f "${SOCK_SHOP_HEALTH_ADDR}"; do
```

Comment:
why did the `-t` change?

Reply:
Something about `timeout` was different between the two. The new one did not like `-t`. I will retry with `-t` because things have evolved.

Reply:
Retried using`-t` and seeing the following:
```
>kc -n px-sock-shop logs load-test-76fb456564-mfb8r -c wait-sock-shop | head -n 20
+ timeout -t 2 curl -f http://orders.px-sock-shop.svc.cluster.local/health
timeout: unrecognized option: t
BusyBox v1.35.0 (2022-08-01 15:14:44 UTC) multi-call binary.

Usage: timeout [-s SIG] [-k KILL_SECS] SECS PROG ARGS

Run PROG. Send SIG to it if it is not gone in SECS seconds.
Default SIG: TERM.If it still exists in KILL_SECS seconds, send KILL.

+ echo 'waiting for http://orders.px-sock-shop.svc.cluster.local/health'
+ sleep 2
waiting for http://orders.px-sock-shop.svc.cluster.local/health
+ timeout -t 2 curl -f http://orders.px-sock-shop.svc.cluster.local/health
timeout: unrecognized option: t
BusyBox v1.35.0 (2022-08-01 15:14:44 UTC) multi-call binary.

Usage: timeout [-s SIG] [-k KILL_SECS] SECS PROG ARGS

Run PROG. Send SIG to it if it is not gone in SECS seconds.
Default SIG: TERM.If it still exists in KILL_SECS seconds, send KILL.
```